scripts/eurostat/gdp/eurostat_gdp.tmcf

```diff
@@ -0,0 +1,70 @@
+Node: E:eurostat_gdp->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcid:Amount_EconomicActivity_GrossDomesticProduction
+observationAbout: C:eurostat_gdp->geo
+observationDate: C:eurostat_gdp->time
+measurementMethod: dcs:EurostatRegionalStatistics
+observationPeriod: "P1M"
+value: C:eurostat_gdp->geo
+unit: dcs:Euro
+
+Node: E:eurostat_gdp->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcid:Amount_EconomicActivity_GrossDomesticProduction
+observationAbout: C:eurostat_gdp->geo
+observationDate: C:eurostat_gdp->time
+measurementMethod: dcs:EurostatRegionalStatistics
+observationPeriod: "P1M"
+value: C:eurostat_gdp->time
+unit: dcs:Euro
```

Comment:
Were these two nodes are mistakenly added?

Reply:
Wow totally missed this. Thanks for the catch!